bitrise.yml

```diff
@@ -0,0 +1,32 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: android
+workflows:
+  ci:
+    steps:
+      - script:
+          title: Provide Instrumentation Tests Artifacts
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                ./gradlew buildIntegrationTestsArtifacts
+      - cache-push@2.0.1: {}
```

Comment:
IIRC, doing a cache push here doesn't help if you're never pulling the cache

Reply:
Yup...I forgot to remove that....added it at some point because something was not working and thought I need to refresh the cache.